### ISSUE TITLE
polymer test fails on Safari with Variants

### ISSUE THREAD

**Issue by e111077** [Polymer/polymer-cli#539]
### Description

When you set up variants in the bower.json and run `polymer test`, Safari explodes with an error. Pasted at the bottom because it's p long. It works fine when the variants bower_components folder is deleted. The problem I assume that it is attempting to run the safari webdriver twice (while it's already running).

### Versions & Environment

- Polymer CLI: 0.18.0-alpha.7
- node: v6.4.0
- Operating System: macOS Sierra 10.12.2

#### Steps to Reproduce
1. Create a variants entry in the bower.json
2. Write some trivial test that will run on wct
3. Have Safari installed
4. Run `polymer test`

#### Expected Results

tests should run for safari

#### Actual Results

Safari tests explode with the following error message

Error message:
```
Error:{
  "state":"unknown error",
  "sessionId":null,
  "hCode":830355877,
  "value":{
    "localizedMessage":"Connection refused",
    "cause":null,
    "stackTrace":[
      {
        "fileName":"PlainSocketImpl.java",
        "nativeMethod":true,
        "methodName":"socketConnect",
        "className":"java.net.PlainSocketImpl",
        "hCode":1566792002,
        "lineNumber":-2,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"AbstractPlainSocketImpl.java",
        "nativeMethod":false,
        "methodName":"doConnect",
        "className":"java.net.AbstractPlainSocketImpl",
        "hCode":495250506,
        "lineNumber":350,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"AbstractPlainSocketImpl.java",
        "nativeMethod":false,
        "methodName":"connectToAddress",
        "className":"java.net.AbstractPlainSocketImpl",
        "hCode":-2104362742,
        "lineNumber":206,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"AbstractPlainSocketImpl.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"java.net.AbstractPlainSocketImpl",
        "hCode":-1864045581,
        "lineNumber":188,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"SocksSocketImpl.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"java.net.SocksSocketImpl",
        "hCode":1780853535,
        "lineNumber":392,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"Socket.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"java.net.Socket",
        "hCode":1700005482,
        "lineNumber":589,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"PlainConnectionSocketFactory.java",
        "nativeMethod":false,
        "methodName":"connectSocket",
        "className":"org.apache.http.conn.socket.PlainConnectionSocketFactory",
        "hCode":1175278798,
        "lineNumber":74,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultHttpClientConnectionOperator.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"org.apache.http.impl.conn.DefaultHttpClientConnectionOperator",
        "hCode":-502593941,
        "lineNumber":141,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"PoolingHttpClientConnectionManager.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"org.apache.http.impl.conn.PoolingHttpClientConnectionManager",
        "hCode":-1370960713,
        "lineNumber":353,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"MainClientExec.java",
        "nativeMethod":false,
        "methodName":"establishRoute",
        "className":"org.apache.http.impl.execchain.MainClientExec",
        "hCode":848422720,
        "lineNumber":380,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"MainClientExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.MainClientExec",
        "hCode":1688513621,
        "lineNumber":236,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ProtocolExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.ProtocolExec",
        "hCode":-642149983,
        "lineNumber":184,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RetryExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.RetryExec",
        "hCode":2007799545,
        "lineNumber":88,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RedirectExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.RedirectExec",
        "hCode":608278487,
        "lineNumber":110,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"InternalHttpClient.java",
        "nativeMethod":false,
        "methodName":"doExecute",
        "className":"org.apache.http.impl.client.InternalHttpClient",
        "hCode":2023045549,
        "lineNumber":184,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"CloseableHttpClient.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.client.CloseableHttpClient",
        "hCode":1628029009,
        "lineNumber":71,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"CloseableHttpClient.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.client.CloseableHttpClient",
        "hCode":1628028993,
        "lineNumber":55,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ApacheHttpClient.java",
        "nativeMethod":false,
        "methodName":"fallBackExecute",
        "className":"org.openqa.selenium.remote.internal.ApacheHttpClient",
        "hCode":652052258,
        "lineNumber":142,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ApacheHttpClient.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.internal.ApacheHttpClient",
        "hCode":-904183858,
        "lineNumber":88,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ProtocolHandshake.java",
        "nativeMethod":false,
        "methodName":"createSession",
        "className":"org.openqa.selenium.remote.ProtocolHandshake",
        "hCode":-960127848,
        "lineNumber":108,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ProtocolHandshake.java",
        "nativeMethod":false,
        "methodName":"createSession",
        "className":"org.openqa.selenium.remote.ProtocolHandshake",
        "hCode":-960127892,
        "lineNumber":64,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"HttpCommandExecutor.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.HttpCommandExecutor",
        "hCode":-1852790188,
        "lineNumber":141,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DriverCommandExecutor.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.service.DriverCommandExecutor",
        "hCode":-633333582,
        "lineNumber":82,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RemoteWebDriver.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.RemoteWebDriver",
        "hCode":1767885088,
        "lineNumber":601,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RemoteWebDriver.java",
        "nativeMethod":false,
        "methodName":"startSession",
        "className":"org.openqa.selenium.remote.RemoteWebDriver",
        "hCode":-1182648233,
        "lineNumber":241,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RemoteWebDriver.java",
        "nativeMethod":false,
        "methodName":"<init>",
        "className":"org.openqa.selenium.remote.RemoteWebDriver",
        "hCode":2001602308,
        "lineNumber":128,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"SafariDriver.java",
        "nativeMethod":false,
        "methodName":"<init>",
        "className":"org.openqa.selenium.safari.SafariDriver",
        "hCode":-1792744904,
        "lineNumber":62,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"SafariDriver.java",
        "nativeMethod":false,
        "methodName":"<init>",
        "className":"org.openqa.selenium.safari.SafariDriver",
        "hCode":-1792744913,
        "lineNumber":53,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"NativeConstructorAccessorImpl.java",
        "nativeMethod":true,
        "methodName":"newInstance0",
        "className":"sun.reflect.NativeConstructorAccessorImpl",
        "hCode":-917798116,
        "lineNumber":-2,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"NativeConstructorAccessorImpl.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"sun.reflect.NativeConstructorAccessorImpl",
        "hCode":-1864225098,
        "lineNumber":62,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DelegatingConstructorAccessorImpl.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"sun.reflect.DelegatingConstructorAccessorImpl",
        "hCode":-2122307259,
        "lineNumber":45,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"Constructor.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"java.lang.reflect.Constructor",
        "hCode":-1319859919,
        "lineNumber":423,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultDriverProvider.java",
        "nativeMethod":false,
        "methodName":"callConstructor",
        "className":"org.openqa.selenium.remote.server.DefaultDriverProvider",
        "hCode":-2096353668,
        "lineNumber":103,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultDriverProvider.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"org.openqa.selenium.remote.server.DefaultDriverProvider",
        "hCode":1201680335,
        "lineNumber":97,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultDriverFactory.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"org.openqa.selenium.remote.server.DefaultDriverFactory",
        "hCode":-1365021416,
        "lineNumber":60,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultSession.java",
        "nativeMethod":false,
        "methodName":"call",
        "className":"org.openqa.selenium.remote.server.DefaultSession$BrowserCreator",
        "hCode":2056168471,
        "lineNumber":222,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultSession.java",
        "nativeMethod":false,
        "methodName":"call",
        "className":"org.openqa.selenium.remote.server.DefaultSession$BrowserCreator",
        "hCode":2056168458,
        "lineNumber":209,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"FutureTask.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"java.util.concurrent.FutureTask",
        "hCode":-820313554,
        "lineNumber":266,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultSession.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"org.openqa.selenium.remote.server.DefaultSession$1",
        "hCode":-255143467,
        "lineNumber":176,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ThreadPoolExecutor.java",
        "nativeMethod":false,
        "methodName":"runWorker",
        "className":"java.util.concurrent.ThreadPoolExecutor",
        "hCode":-1285150184,
        "lineNumber":1142,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ThreadPoolExecutor.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"java.util.concurrent.ThreadPoolExecutor$Worker",
        "hCode":-242190645,
        "lineNumber":617,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"Thread.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"java.lang.Thread",
        "hCode":-82946320,
        "lineNumber":745,
        "class":"java.lang.StackTraceElement"
      }
    ],
    "suppressed":[

    ],
    "message":"Connection refused",
    "hCode":1762351813,
    "class":"java.net.ConnectException",
    "screen":null
  },
  "class":"org.openqa.selenium.remote.Response",
  "status":13
},
{
  "state":"unknown error",
  "sessionId":null,
  "hCode":1403970263,
  "value":{
    "localizedMessage":"Connection refused",
    "cause":null,
    "stackTrace":[
      {
        "fileName":"PlainSocketImpl.java",
        "nativeMethod":true,
        "methodName":"socketConnect",
        "className":"java.net.PlainSocketImpl",
        "hCode":1566792002,
        "lineNumber":-2,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"AbstractPlainSocketImpl.java",
        "nativeMethod":false,
        "methodName":"doConnect",
        "className":"java.net.AbstractPlainSocketImpl",
        "hCode":495250506,
        "lineNumber":350,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"AbstractPlainSocketImpl.java",
        "nativeMethod":false,
        "methodName":"connectToAddress",
        "className":"java.net.AbstractPlainSocketImpl",
        "hCode":-2104362742,
        "lineNumber":206,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"AbstractPlainSocketImpl.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"java.net.AbstractPlainSocketImpl",
        "hCode":-1864045581,
        "lineNumber":188,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"SocksSocketImpl.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"java.net.SocksSocketImpl",
        "hCode":1780853535,
        "lineNumber":392,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"Socket.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"java.net.Socket",
        "hCode":1700005482,
        "lineNumber":589,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"PlainConnectionSocketFactory.java",
        "nativeMethod":false,
        "methodName":"connectSocket",
        "className":"org.apache.http.conn.socket.PlainConnectionSocketFactory",
        "hCode":1175278798,
        "lineNumber":74,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultHttpClientConnectionOperator.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"org.apache.http.impl.conn.DefaultHttpClientConnectionOperator",
        "hCode":-502593941,
        "lineNumber":141,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"PoolingHttpClientConnectionManager.java",
        "nativeMethod":false,
        "methodName":"connect",
        "className":"org.apache.http.impl.conn.PoolingHttpClientConnectionManager",
        "hCode":-1370960713,
        "lineNumber":353,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"MainClientExec.java",
        "nativeMethod":false,
        "methodName":"establishRoute",
        "className":"org.apache.http.impl.execchain.MainClientExec",
        "hCode":848422720,
        "lineNumber":380,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"MainClientExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.MainClientExec",
        "hCode":1688513621,
        "lineNumber":236,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ProtocolExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.ProtocolExec",
        "hCode":-642149983,
        "lineNumber":184,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RetryExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.RetryExec",
        "hCode":2007799545,
        "lineNumber":88,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RedirectExec.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.execchain.RedirectExec",
        "hCode":608278487,
        "lineNumber":110,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"InternalHttpClient.java",
        "nativeMethod":false,
        "methodName":"doExecute",
        "className":"org.apache.http.impl.client.InternalHttpClient",
        "hCode":2023045549,
        "lineNumber":184,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"CloseableHttpClient.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.client.CloseableHttpClient",
        "hCode":1628029009,
        "lineNumber":71,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"CloseableHttpClient.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.apache.http.impl.client.CloseableHttpClient",
        "hCode":1628028993,
        "lineNumber":55,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ApacheHttpClient.java",
        "nativeMethod":false,
        "methodName":"fallBackExecute",
        "className":"org.openqa.selenium.remote.internal.ApacheHttpClient",
        "hCode":652052258,
        "lineNumber":142,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ApacheHttpClient.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.internal.ApacheHttpClient",
        "hCode":-904183858,
        "lineNumber":88,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ProtocolHandshake.java",
        "nativeMethod":false,
        "methodName":"createSession",
        "className":"org.openqa.selenium.remote.ProtocolHandshake",
        "hCode":-960127848,
        "lineNumber":108,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ProtocolHandshake.java",
        "nativeMethod":false,
        "methodName":"createSession",
        "className":"org.openqa.selenium.remote.ProtocolHandshake",
        "hCode":-960127892,
        "lineNumber":64,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"HttpCommandExecutor.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.HttpCommandExecutor",
        "hCode":-1852790188,
        "lineNumber":141,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DriverCommandExecutor.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.service.DriverCommandExecutor",
        "hCode":-633333582,
        "lineNumber":82,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RemoteWebDriver.java",
        "nativeMethod":false,
        "methodName":"execute",
        "className":"org.openqa.selenium.remote.RemoteWebDriver",
        "hCode":1767885088,
        "lineNumber":601,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RemoteWebDriver.java",
        "nativeMethod":false,
        "methodName":"startSession",
        "className":"org.openqa.selenium.remote.RemoteWebDriver",
        "hCode":-1182648233,
        "lineNumber":241,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"RemoteWebDriver.java",
        "nativeMethod":false,
        "methodName":"<init>",
        "className":"org.openqa.selenium.remote.RemoteWebDriver",
        "hCode":2001602308,
        "lineNumber":128,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"SafariDriver.java",
        "nativeMethod":false,
        "methodName":"<init>",
        "className":"org.openqa.selenium.safari.SafariDriver",
        "hCode":-1792744904,
        "lineNumber":62,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"SafariDriver.java",
        "nativeMethod":false,
        "methodName":"<init>",
        "className":"org.openqa.selenium.safari.SafariDriver",
        "hCode":-1792744913,
        "lineNumber":53,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"NativeConstructorAccessorImpl.java",
        "nativeMethod":true,
        "methodName":"newInstance0",
        "className":"sun.reflect.NativeConstructorAccessorImpl",
        "hCode":-917798116,
        "lineNumber":-2,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"NativeConstructorAccessorImpl.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"sun.reflect.NativeConstructorAccessorImpl",
        "hCode":-1864225098,
        "lineNumber":62,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DelegatingConstructorAccessorImpl.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"sun.reflect.DelegatingConstructorAccessorImpl",
        "hCode":-2122307259,
        "lineNumber":45,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"Constructor.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"java.lang.reflect.Constructor",
        "hCode":-1319859919,
        "lineNumber":423,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultDriverProvider.java",
        "nativeMethod":false,
        "methodName":"callConstructor",
        "className":"org.openqa.selenium.remote.server.DefaultDriverProvider",
        "hCode":-2096353668,
        "lineNumber":103,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultDriverProvider.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"org.openqa.selenium.remote.server.DefaultDriverProvider",
        "hCode":1201680335,
        "lineNumber":97,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultDriverFactory.java",
        "nativeMethod":false,
        "methodName":"newInstance",
        "className":"org.openqa.selenium.remote.server.DefaultDriverFactory",
        "hCode":-1365021416,
        "lineNumber":60,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultSession.java",
        "nativeMethod":false,
        "methodName":"call",
        "className":"org.openqa.selenium.remote.server.DefaultSession$BrowserCreator",
        "hCode":2056168471,
        "lineNumber":222,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultSession.java",
        "nativeMethod":false,
        "methodName":"call",
        "className":"org.openqa.selenium.remote.server.DefaultSession$BrowserCreator",
        "hCode":2056168458,
        "lineNumber":209,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"FutureTask.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"java.util.concurrent.FutureTask",
        "hCode":-820313554,
        "lineNumber":266,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"DefaultSession.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"org.openqa.selenium.remote.server.DefaultSession$1",
        "hCode":-255143467,
        "lineNumber":176,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ThreadPoolExecutor.java",
        "nativeMethod":false,
        "methodName":"runWorker",
        "className":"java.util.concurrent.ThreadPoolExecutor",
        "hCode":-1285150184,
        "lineNumber":1142,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"ThreadPoolExecutor.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"java.util.concurrent.ThreadPoolExecutor$Worker",
        "hCode":-242190645,
        "lineNumber":617,
        "class":"java.lang.StackTraceElement"
      },
      {
        "fileName":"Thread.java",
        "nativeMethod":false,
        "methodName":"run",
        "className":"java.lang.Thread",
        "hCode":-82946320,
        "lineNumber":745,
        "class":"java.lang.StackTraceElement"
      }
    ],
    "suppressed":[

    ],
    "message":"Connection refused",
    "hCode":1356063820,
    "class":"java.net.ConnectException",
    "screen":null
  },
  "class":"org.openqa.selenium.remote.Response",
  "status":13
}
```

**Comment by FredKSchott** [comment]
@e111077 can you reproduce by running the latest alpha version of wct (instead of polymer test)?

**Comment by e111077** [comment]
Cannot repro with WCT prerelease.3

**Comment by FredKSchott** [comment]
Huh, I wouldn't have expected that. @rictic can you take a look?

**Comment by e111077** [comment]
Problem resolved after installing iojs 3.3.1 (which wasn't installed before) and upgrading node to v7. I believe that it was stemming from a node-bunyan warning `Error: Cannot find module './build/Release/DTraceProviderBindings'`.
